README.md

```diff
@@ -110,6 +111,14 @@ WILL NOT have any effect on its downstream users.
 
 [`.cargo/config.toml`]: https://doc.rust-lang.org/cargo/reference/config.html
 
+### Raw Linux syscall support
+
+Currently the `linux_raw` backend supports only targets with stabilized `asm!` macro,
+i.e. `arm`, `aarch64`, `loongarch64`, `riscv32`, `riscv64`, `x86`, and `x86_64`.
```

Comment:
Is there a reason s390x (the only other stable `asm!` target) is excluded? I'm totally fine not supporting it (similar to `x86` and 32-bit `arm`) if it's challenging in any way.

Reply:
`asm!` was stabilized for it only in Rust 1.84. I will try to add it a bit later.